tsl/src/hypercore/utils.c

```diff
@@ -54,6 +57,21 @@ hypercore_set_am(const RangeVar *rv)
 		};
 
 		recordDependencyOn(&depender, &referenced, DEPENDENCY_NORMAL);
+
+		/*
+		 * Update the tuple for the compressed chunk and disable autovacuum on
+		 * it. This requires locking the relation (to prevent changes to the
+		 * definition), but it is sufficient to take an access share lock.
+		 */
+		Chunk *chunk = ts_chunk_get_by_relid(relid, true);
+		Chunk *cchunk = ts_chunk_get_by_id(chunk->fd.compressed_chunk_id, true);
+		Relation compressed_rel = table_open(cchunk->table_id, AccessShareLock);
+
+		/* We use makeInteger since makeBoolean does not exist prior to PG15 */
```

Comment:
What about adding `makeBoolean` to `compat.h` and make the code clearer?

Reply:
IMHO, it is better to keep this code using `makeInteger` and make the necessary changes when we deprecate PG15.

Reply:
You meant when we deprecate PG14, right? Just wonder that if we don't do it now probably we'll never do it later. But it is a minor thing and up to you!